src/screens/Ticketing/FareProducts/use-recent-fare-contracts.ts

```diff
@@ -107,11 +135,6 @@ const mapBackendRecentFareContracts = (
     recentFareContract.zones.slice(-1)[0],
   );
 
-  const userProfilesWithCount = mapUsers(
-    recentFareContract.users,
-    userProfiles,
-  );
-
   if (
     !preassignedFareProduct ||
     !fromTariffZone ||
```

Comment:
Maybe the check on `preassignedFareProduct` and `userProfilesWithCount?.length` is redundant here as it is already checked further up?

Reply:
Fixed now, please take a look ;)